modules/tinymce/src/plugins/table/main/ts/api/Settings.ts

```diff
@@ -44,6 +44,17 @@ const isPercentagesForced = (editor: Editor): boolean => editor.getParam('table_
 const isPixelsForced = (editor: Editor): boolean => editor.getParam('table_responsive_width') === false;
 const getToolbar = (editor: Editor): string => editor.getParam('table_toolbar', defaultTableToolbar);
 
+
+const getTableHeaderType = (editor: Editor): string => {
+  const value = editor.getParam('table_header_type', 'auto', 'string');
+  const validValues = [ 'thead', 'ths', 'both', 'auto' ];
```

Comment:
Should this be `th` as `ths` isn't very clear (or at least to me)?

Reply:
Note: discussing this with the team tomorrow

Reply:
Have changed this